src/content-script.ts

```diff
@@ -124,38 +149,83 @@ async function main() {
 
 				replyCategoriesDetailsMenu.appendChild(
 					createElement("ul", {
+						"aria-labelled-by": "repository-replies-label",
 						children: [
-							createElement("li", { children: [button], role: "none" }),
+							createElement("li", {
+								children: [button],
+								role: "none",
+							}),
 						],
-						role: "none",
+						className: "js-saved-reply-menu ActionListWrap",
+						role: "list",
 					}),
 				);
 			}
 
-			replyCategoriesDetailsMenu.appendChild(
-				createElement("a", {
-					children: [
-						createElement("span", {
-							children: ["Create a new repository reply..."],
-							className: "select-menu-item-text",
-						}),
-					],
-					className: "select-menu-item select-menu-action",
-					href: `https://github.com/${userOrOrganization}/${repository}/edit/${defaultBranch}/.github/replies.yml`,
-					role: "menuitem",
-					target: "_blank",
-				}),
-			);
+			// 8. Add a second button at the bottom of the modal for adding more
+			// TODO: thanks for the heads up @keithamus :)
+			// https://github.com/primer/view_components/pull/2364
+			for (const modal of Array.from(
+				document.querySelectorAll<HTMLElement>(
+					"modal-dialog#saved_replies_menu_new_comment_field-dialog",
+				),
+			)) {
+				// Also, because the modal is by default too tiny, let's make it bigger
+				modal.style.height = "100%";
+				modal.style.maxHeight = "calc(100vh - 5rem)";
```

Comment:
https://github.com/primer/view_components/blob/97f03393977c9c82141eb5255fa3fd619a95e935/app/components/primer/alpha/dialog.rb#L26-L31 these are the sizes defined in the design system. So maybe add the class `.Overlay--size-xlarge`